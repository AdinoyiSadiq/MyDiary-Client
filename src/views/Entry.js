import React from 'react';
import Main from '../components/layout/Main';

const Entry = (props) => (
  <div>
    <Main {...props}>
      This is the Entry view
    </Main>
  </div>
);

export default Entry;