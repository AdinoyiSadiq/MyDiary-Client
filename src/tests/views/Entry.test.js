import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Entry from '../../views/Entry';

let wrapped;

beforeEach(() => {
  wrapped = shallow(<Entry />);
});

describe('Entry UI', () => {
  describe('render features', () => {
    test('view should render as expected', () => {
      const tree = toJson(wrapped);
      expect(tree).toMatchSnapshot();
    });
  });
});