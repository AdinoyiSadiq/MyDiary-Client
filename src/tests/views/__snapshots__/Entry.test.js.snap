// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Entry UI render features view should render as expected 1`] = `
<div
  className="entryContent"
>
  <Main>
    <Connect(SingleEntry) />
  </Main>
</div>
`;
